<ui version="4.0" >
 <class>ChannelListItemClass</class>
 <widget class="QFrame" name="ChannelListItemClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>67</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>67</height>
   </size>
  </property>
  <property name="frameShape" >
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow" >
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth" >
   <number>0</number>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>7</number>
   </property>
   <property name="topMargin" >
    <number>6</number>
   </property>
   <property name="rightMargin" >
    <number>7</number>
   </property>
   <property name="bottomMargin" >
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="imageLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>51</width>
       <height>51</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>51</width>
       <height>51</height>
      </size>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <property name="spacing" >
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="titleLabel" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>TextLabel_DT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="descriptionLabel" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>44</height>
        </size>
       </property>
       <property name="text" >
        <string>TextLabel_DT</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="addChannelButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>80</width>
       <height>25</height>
      </size>
     </property>
     <property name="accessibleName" >
      <string>bigRectButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="removeChannelButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>80</width>
       <height>25</height>
      </size>
     </property>
     <property name="accessibleName" >
      <string>bigRectButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
